app.py

```diff
@@ -1,15 +1,11 @@
 from flask import Flask
 
 # Flask
-app = Flask(__name__)
+
+app = Flask(__name__, template_folder="c:/pts_feedback/views")
```

Comment:
Remember some of us run this app on a Mac/linux machine. Use relative paths.

Reply:
`views` would be enough.

Reply:
Yeah, kind of knew there is some better way to do this. I tried set-variable first but it didn't work - but good that this works, I'll edit this to next version